js/bulkhead-callback.js

```diff
@@ -279,16 +283,73 @@ var bulkheadCallBack = (function() {
         return match;
     };
 
+    var __validateEditorContent_BulkheadStep = function(content) {
+        var match = false;
+        try {
+            var pattern = "@Bulkhead\\(50\\)\\s*public Service serviceForVFA";
```

Comment:
You need to add the beginning boundary as user could type in other junks before typing in @Bulkhead(50), eg.

      return null;
    });
  }
myMisBehavior();
@bulkhead(50)
public Service serviceForVFA